keras/layers/convolutional.py

```diff
@@ -1449,7 +1464,17 @@ class ZeroPadding2D(Layer):
     # Arguments
         padding: tuple of int (length 2)
             How many zeros to add at the beginning and end of
-            the 2 padding dimensions (axis 3 and 4).
+            the 2 padding dimensions (rows and cols) symmetrically.
+            Setting padding to the values different from (1, 1)
+            overrides top_pad, bottom_pad; and left_pad, right_pad arguments.
```

Comment:
Really not a fan of this API. Maybe we should go with a single polymorphic argument, `padding`. If tuple of length 2, it's symmetric padding. If length 4, it's in order `top_pad`, `bottom_pad`, `left_pad`, `right_pad `. It can also be dictionary with keys `top_pad`, `bottom_pad`, and `left_pad`, `right_pad`; any missing keys defaults to 0. Any unexpected key or tuple of unexpected size triggers a helpful exception.

What do you think?